pkgs/tools/X11/xpra/default.nix

```diff
@@ -70,11 +70,13 @@ let
   '';
 in buildPythonApplication rec {
   pname = "xpra";
-  version = "4.4.6";
+  version = "5.0.9";
 
-  src = fetchurl {
-    url = "https://xpra.org/src/${pname}-${version}.tar.xz";
-    hash = "sha256-BWf3nypfSrYCzpJ0OfBkecoHGbG1lEgu5jLZhfkIejQ=";
+  src = fetchFromGitHub {
+    owner = "Xpra-org";
+    repo = pname;
```

Comment:
We’re generally moving away from doing this with the `finalAttrs` migration. Could you set this to `"xpra"` specifically instead?